src/components/homePageLayout/aboutus/aboutus.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import HomePageTitle from '../HomePageTitle/homePageTitle';
+import {Card,Row,Col,Image,Jumbotron} from 'react-bootstrap';
+import LeftImage from './aboutusLeft.png';
+import './aboutus.scss';
+import MoreButton from '../MoreButton/MoreButton';
+import left from './left.png';
+import middle from './middle.png';
+import right from './right.png';
+import CommonCol from './commonCard';
+
+const AboutUS = () => {
+	return (
+		<section className="homePageAboutUsSection" >
+			<HomePageTitle sectionTitle="About US" sectionSubTitle="关于我们" link="/about-us"/>
+			<Card className="mb-3" style={{borderRadius:'0px', paddingLeft: '6%',border:'none'}}>
+				<Row className="no-gutters justify-content-enter" style={{paddingRight:'6%', paddingTop:'3vw'}}>
```

Comment:
you don't need to create a row here. Check https://react-bootstrap.github.io/components/cards/#card-groups